Gemfile.lock

```diff
@@ -106,7 +106,7 @@ GEM
       rest-client (~> 2.0)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
-    govuk_app_config (2.7.0)
+    govuk_app_config (2.7.1)
```

Comment:
Does this need bumping?

Reply:
Thanks for the review @andysellick! 

Doesn't need bumping as part of this change but may as well as it's only a minor patch and was included when I bundled.